lib/streamlit/server/routes.py

```diff
@@ -55,6 +60,17 @@ def set_extra_headers(self, path):
         else:
             self.set_header("Cache-Control", "public")
 
+    def parse_url_path(self, url_path: str) -> str:
+        url_parts = url_path.split("/")
+
+        # TODO(vdonato): Maybe clean this up if we decide to tweak the spec so
+        # that there's a separator between the baseUrlPath and page name.
```

Comment:
Does the `url_path` contain the full URL path that is used in the frontend (e..g incl. `baseUrlPath`)?  Or how do we deal with `baseUrlPath` currently?

Reply:
`baseUrlPath` isn't included in `url_path` at this point since it gets matched earlier in the routing process